### PR TITLE
Implement sticky results bar in javascript.

### PR DESCRIPTION
**Pull request by beshaya**
Avoids needing to change overflow.

### CODE REVIEW

File: src/pages/Calculator.tsx [hunk 1]
```diff
@@ -26,6 +26,14 @@ const localStorage = window.localStorage
 const FORM_STATE_KEY = 'formData'
 
 export const Calculator = (): React.ReactElement => {
+  // Mount
+  useEffect(() => window.addEventListener('scroll', scrollListener, true), [])
+  // Unmount
+  useEffect(
```
Comment (dominic) [inline src/pages/Calculator.tsx:32]: One too many useEffects; just return the unmount handler from the first call

https://reactjs.org/docs/hooks-reference.html

File: src/pages/Calculator.tsx [hunk 2]
```diff
@@ -207,16 +215,38 @@ export const Calculator = (): React.ReactElement => {
...
         <Col lg={{ span: 8, offset: 4 }}>
           <ExplanationCard points={points} />
         </Col>
       </Row>
     </div>
   )
 }
+
+// Sets #pointsRow to position: fixed if it would not be on the screen.
+function scrollListener(): void {
```
Comment (dominic) [inline src/pages/Calculator.tsx:233]: New function syntax preferred: `const scrollListener = ...`

**Comment by dominic** [comment]
This is less helpful when scrolling down the calculator right now - maybe only enable stickiness if there's a point total available as well?

image

File: src/pages/Calculator.tsx (hunk 1 above)
Comment (beshaya) [reply src/pages/Calculator.tsx]: Done!

**Pull request merged**